config.py

```diff
@@ -759,7 +759,35 @@
  	        'Немає АД 2',
         ],
         hidden_category = 'Приховані_категорії',
-    )
+    ),
+
+    hi = dict(
```

Comment:
Are there any citation_needed_templates to add? Or do you just literally use `{{ cn }}` and others from enwiki on these two wikis?